bin/check.sh

```diff
@@ -65,7 +43,19 @@ _shellcheck_output_format() {
 
 # See discussion in https://github.com/icy/pacapt/pull/59
 _has_shellcheck() {
-  command -v shellcheck >/dev/null 2>&1
+  : "${SHELLCHECK_TAG:=v0.7.2}"
+  if [[ -n "${CI_SHELLCHECK_UPDATE:-}" && "$OSTYPE" =~ linux.* ]]; then
```

Comment:
This script is used by CI system, so it's fine to use bash here

Reply:
The main scripts (`lib/*.sh`) are either asking Bash or Sh. However, it's executed with `sh` by default, and if there is `bash` on the system, it switches over to new shell quickly.  This is great because we don't really need to port all current libraries to POSIX style.

Reply:
Ack!